### ISSUE TITLE
[Question] Does playwright support browser proxy along with selenium grid?

### ISSUE THREAD

**Issue by szezecinski** [microsoft/playwright#16772]
### **TL;DR: playwright doesn't appear to be passing out proxy settings to selenium grid.**

I have been struggling to make MS Playwright work along with selenium Grid 4, apparently playwright doesn’t handle very well proxy settings while working with Selenium grid.
I have tried to make them work together via C# and TS and with docker containers in different OS (Linux and Windows), but I couldn't see proxy settings being applied in none of the tests that I used playwright.

When I used a different set of technologies for instance “C# + RemoteWebDriver” or ”JS + selenium-webdriver”, everything worked as expected. Therefore, I strongly believe that this is not a Selenium Grid or a Docker env issue. Maybe I’m not informing browser capabilities correctly. 

I've created a git repo containing what I'm trying to do + what I have seen so far:

[Overall Git Repo](https://github.com/szezecinski/help-with-playwright-proxy)

[Docker commands to spin up Grid and Node Containers](https://github.com/szezecinski/help-with-playwright-proxy/blob/main/1__create-docker-containers.txt)

--> [Sample test using playwright](https://github.com/szezecinski/help-with-playwright-proxy/blob/main/TestWithPlaywright.cs)
----> [Results of previous test](https://github.com/szezecinski/help-with-playwright-proxy/blob/main/results/playwright_session/session_print.md)

--> [Sample test using remote webdriver](https://github.com/szezecinski/help-with-playwright-proxy/blob/main/TestWithRemoteWebDriver.cs)
-----> [Results of previous test](https://github.com/szezecinski/help-with-playwright-proxy/blob/main/results/remoteWebDriver_session/session_print.md)

Would you please help me to make this work with C# and TS?

**Comment by szezecinski** [comment]
@yury-s  , do you have any feedback for this item?

**Comment by fde-lmag** [comment]
Thnx
... and Damn

i tried to pull some information out of the Team by opening a ticket
https://github.com/microsoft/playwright/issues/21613
but... no luck at all!

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your involvement. This issue was closed due to limited engagement (upvotes/activity), lack of recent activity, and insufficient actionability. To maintain a manageable database, we prioritize issues based on these factors.

If you disagree with this closure, please open a new issue and reference this one. More support or clarity on its necessity may prompt a review. Your understanding and cooperation are appreciated.
